bundles/ui/org.eclipse.smarthome.ui.classic/snippets/group_expanded.html

```diff
@@ -0,0 +1,2 @@
+%children%
+</ul></div>
```

Comment:
Are these closing tags really correct? They are not opened anywhere... Looks like a left-over from copying the frame.html?